site/src/atomics/spacing.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Pattern
+
+- `<cssproperty>-<size>-<screensize>`
+- Properties: `margin, padding, <margin|padding>-<left|right|top|bottom>, <margin|padding>-inline, <margin|padding>-block`
```

Comment:
Would this look more simple if we just listed the properties and removed the regexp type syntax?